builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -1063,3 +1066,73 @@ def loadBuildModules(buildModuleUrl){
     }
   }
 }
+
+def servelessDeploy(env, envBucketKey){
+    echo "which env: $env"
+    echo "which bucket key: $envBucketKey"
+  try {
+    sh "serverless deploy --stage ${env} -v --bucket ${configLoader.AWS.S3[envBucketKey]} --force > output.log"
+    return "success"
+  } catch (ex) {
+    echo "Serverless deployment failed due to $ex"
+  }
+
+  def outputLog = readFile('output.log').trim()
+  echo "serverless deployment log $outputLog"
+  return outputLog
+}
+
+
+def isExistingLogGroup (groupName){
+	def isExistingLogGrp = false
+	def filterLogs = sh (
+		script: "aws logs describe-log-groups --log-group-name-prefix $groupName --output json --profile cloud-api",
```

Comment:
We're now creating a new aws profile with each build & cleaning it up at the end. Please use the same profile - $credsId here.

Reply:
Made the changes